Grand.Web/Areas/Admin/Views/Common/CustomRobotTxt.cshtml

```diff
@@ -0,0 +1,79 @@
+﻿@model Editor
+@inject IPageHeadBuilder pagebuilder
+@{
+    //page title
+    ViewBag.Title = T("Admin.System.CustomRobotTxt").Text;
+}
+@{
+    pagebuilder.AddScriptParts(ResourceLocation.Head, "~/content/codemirror/codemirror.js", excludeFromBundle: true);
+    pagebuilder.AddScriptParts(ResourceLocation.Head, "~/content/codemirror/mode/css/css.js", excludeFromBundle: true);
+    pagebuilder.AddScriptParts(ResourceLocation.Head, "~/content/codemirror/hint/show-hint.js", excludeFromBundle: true);
+    pagebuilder.AddScriptParts(ResourceLocation.Head, "~/content/codemirror/hint/css-hint.js", excludeFromBundle: true);
+    pagebuilder.AddScriptParts(ResourceLocation.Head, "~/content/codemirror/edit/matchbrackets.js", excludeFromBundle: true);
```

Comment:
As you correctly noticed, it's not possible to implement hints, so why have you added js and CSS resources for hints? :) It's enough to keep only codemirror.js and codemirror.css file.